pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -30,6 +31,12 @@ const (
 	ignoredEncodedPolicyString = ""
 )
 
+var testRand *rand.Rand
+
+func init() {
```

Comment:
Are the tests still repeatable so they always repro a problem?

Reply:
In some cases the "rand" is provided by the test library in others, we'd initialize a new Rand instance in every test. This change makes sure we use one per test run, so if we log the seed, we can (at least) reproduce the bits where this package level Rand instance was used. However, I don't think everything else is as repeatable as we'd like it to be.